src/libs/API.js

```diff
@@ -55,18 +71,6 @@ function addDefaultValuesToParameters(command, parameters) {
     const finalParameters = {...parameters};
 
     if (isAuthTokenRequired(command) && !parameters.authToken) {
-        // If we end up here with no authToken it means we are trying to make an API request before we are signed in.
-        // In this case, we should cancel the current request by pausing the queue and clearing the remaining requests.
-        if (!authToken) {
-            redirectToSignIn();
-
-            LogUtil.info('A request was made without an authToken', false, {command, parameters});
-            Network.pauseRequestQueue();
-            Network.clearRequestQueue();
-            Network.unpauseRequestQueue();
-            return;
-        }
```

Comment:
@nickmurray47 comment makes me wonder if we should add a test for this. But maybe it is already covered.

Reply:
I guess the manual test would just be...

- Mess up the `authToken` and make sure you are reauthenticated
- Mess up the `credentials` and `authToken` and make sure you are logged out

Reply:
I like adding both of those!